src/Native/Unix/System.Net.Http.Native/pal_versioninfo.cpp

```diff
@@ -94,12 +97,9 @@ extern "C" int32_t HttpNative_GetCurlVersionInfo(int32_t* age,
     *age = versionInfo->age;
     *supportsSsl = (versionInfo->features & CURL_VERSION_SSL) == CURL_VERSION_SSL;
     *supportsAutoDecompression = (versionInfo->features & CURL_VERSION_LIBZ) == CURL_VERSION_LIBZ;
-    *supportsHttp2Multiplexing =
-#if HAVE_CURLPIPE_MULTIPLEX
-        (versionInfo->features & CURL_VERSION_HTTP2) == CURL_VERSION_HTTP2;
-#else
-        0;
-#endif
+    *supportsHttp2Multiplexing = 
```

Comment:
Actually, I think we can delete the whole HttpNative_GetCurlVersionInfo function at this point... I believe it's dead code, replaced by the individual functions like HttpNative_GetSupportsHttp2Multiplexing.  This is confirmed by the TODO at https://github.com/dotnet/corefx/pull/17904/files#diff-c340184a1ed09fe1e5a2c790d293204cR74 and by a search showing that the only occurrence of HttpNative_GetCurlVersionInfo is this definition.